File: app/src/main/java/com/example/ryadar/tablayoutmaterialdesign/FragmentTwo.java
package com.example.ryadar.tablayoutmaterialdesign;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.ViewGroup;

public class FragmentTwo extends Fragment {


    @Override
    public android.view.View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        android.view.View rootView = inflater.inflate(R.layout.page_two, container, false);
    return rootView;
}

}

File: app/src/main/java/com/example/ryadar/tablayoutmaterialdesign/kt/FragmentTwo.kt
package com.example.ryadar.tablayoutmaterialdesign.kt

 import android.os.Bundle
 import android.support.v4.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import com.example.ryadar.tablayoutmaterialdesign.R

class FragmentTwo :Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View?{
        return inflater.inflate(R.layout.page_two, container, false)
}
}